tortoise/algorithm.go

```diff
@@ -231,12 +288,24 @@ type DecodedBallot struct {
 	minHint types.LayerID
 }
 
-// DecodeBallot decodes ballot if it wasn't processed earlier.
+// decodeBallot decodes ballot if it wasn't processed earlier.
```

Comment:
Why small caps when the function is public?

Reply:
it is a leftover after refactoring. reverted